src/Tribe/Assets.php

```diff
@@ -116,7 +111,7 @@ public function register_in_wp( $assets = null ) {
 	 * @param string|array $groups Which groups will be enqueued.
 	 */
 	public function enqueue_group( $groups ) {
-		$assets = $this->get();
+		$assets = $this->get( null, false );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space

<sub>hash: 4598eab6ec7ab944b8fbd06773fa2082</sub>